app/controllers/homepage_controller.rb

```diff
@@ -0,0 +1,7 @@
+class HomepageController < ApplicationController
+  def home
+    @books = Book.order(ranking: :desc).limit(10)
```

Comment:
If the limit change, you have to change it in three lines, think of the way to avoid it. Hint: use constant instead.